utils/kubernetes/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .kube_state_processor import KubeStateProcessor  # noqa: F401
```

Comment:
That was the idea, along with removing non-stateful methods from `Kubeutil` into another module for clarity. I'd open a separate PR for that